app/src/main/res/layout/onboarding_inactive_tabs_cfr.xml

```diff
@@ -35,7 +35,7 @@
             android:maxWidth="200dp"
             android:text="@string/tab_tray_inactive_onboarding_message"
             android:textAppearance="@style/Body16TextStyle"
-            android:textColor="@color/contrast_text_light_theme"
+            android:textColor="@color/fx_mobile_private_text_color_primary"
```

Comment:
@color/fx_mobile_private_text_color_primary maps to @color/contrast_text_light_theme -> @color/photonLightGrey05. I think it makes sense to **private** primary text color here since these onboarding layout have the same private (purple) theme.

https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/res/values/colors.xml#119